engines/scumm/he/logic/football.cpp

```diff
@@ -472,7 +472,8 @@ int LogicHEfootball2002::getPlaybookFiles(int32 *args) {
 
 	// Now store the result in an array
 	int array = _vm->setupStringArray(output.size());
-	strcpy((char *)_vm->getStringAddress(array), output.c_str());
+	Common::strlcpy((char *)_vm->getStringAddress(array), output.c_str(), output.size() + 1);
+	_vm->getResourceSize(rtString, array);
```

Comment:
Is this addition intentional?

Reply:
Heya, thanks for going over this PR.
I think this is intentional, since ScummEngine_v72he::setupStringArray() also adds +1 to the size argument inside the function. And Common::strlcpy will also reserve one char for the termination `\0`. So, it should work out precisely. But I have to admit this makes my head spin a bit right now. Maybe I just copy the code over to some target that I actually have and test it...

Reply:
So, I've tested it with Putt-Putt travels through time (just copied over those lines) and it seems fine...